src/status_im/contexts/profile/contact/header/view.cljs

```diff
@@ -0,0 +1,38 @@
+(ns status-im.contexts.profile.contact.header.view
+  (:require [clojure.string :as string]
+            [quo.core :as quo]
+            [quo.foundations.colors :as colors]
+            [quo.theme :as quo.theme]
+            [react-native.core :as rn]
+            [status-im.common.scalable-avatar.view :as avatar]
+            [status-im.contexts.profile.contact.header.style :as style]
+            [status-im.contexts.profile.utils :as profile.utils]
+            [utils.re-frame :as rf]))
+
+(defn- f-view
+  [{:keys [theme scroll-y]}]
+  (let [{:keys [public-key customization-color
+                emoji-hash]
+         :as   profile}     (rf/sub [:contacts/current-contact])
+        customization-color (or customization-color :blue)
+        full-name           (profile.utils/displayed-name profile)
+        bio                 (:bio profile)
+        emoji-string        (string/join emoji-hash)
+        profile-picture     (profile.utils/photo profile)
+        online?             (rf/sub [:visibility-status-updates/online? public-key])]
+    [rn/view {:style style/header-container}
+     [rn/view {:style style/avatar-wrapper}
+      [avatar/view
+       {:scroll-y            scroll-y
+        :full-name           full-name
+        :online?             online?
+        :profile-picture     profile-picture
+        :border-color        (colors/theme-colors colors/white colors/neutral-95 theme)
+        :customization-color customization-color}]]
+     [quo/page-top
+      {:title            full-name
+       :description      :text
+       :description-text bio
+       :emoji-dash       emoji-string}]]))
+
+(def view (quo.theme/with-theme f-view))
```

Comment:
`f-view` was something we use for functional components handling. we should move away from that pattern, it can be `view-internal` instead. however I believe @flexsurfer want's to move away from with-theme hoc, so better to make use of `use-theme` hook instead directly in the `let` var bindings for the view 👍